README.md

```diff
@@ -85,7 +85,7 @@ There is **experimental** support for using `deno-sqlite` in the browser. You
 can generate a browser compatible module by running:
 
 ```bash
-deno bundle --import-map browser/import_map.json browser/mod.ts [output_bundle_path]
+deno run --allow-run --allow-env --allow-read --allow-write ./browser/bundle.js
```

Comment:
I think we can look down permissions a bit more here?

`--allow-run` isn’t required, and we can scope reading to the repo and writing to just the bundle?